spec/features/user_can_delete_stations_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe "User can delete station" do
+  it "and click on button" do
+
+    Station.create!(name: "Galvanize", dock_count: 10, city: "Denver", installation_date: "11/08/17")
+
+    Station.create!(name: "Turing", dock_count: 10, city: "Denver", installation_date: "11/08/17")
+
+    visit '/stations/1'
+    click_on("delete")
+
+    expect(current_path).to eq('/stations')
```

Comment:
Would like some more assertions here. You've gone through the process of creating two stations. Would recommend asserting that you still see the one that you didn't delete, and that you don't see the one that you did. You don't have to check every attribute; name would probably be enough.